src/fu-util-common.c

```diff
@@ -1776,3 +1776,24 @@ fu_util_send_report (SoupSession *soup_session,
 	/* success */
 	return TRUE;
 }
+
+gint
+fu_util_sort_devices_by_flags (gconstpointer a, gconstpointer b)
```

Comment:
I think the convention for this kind of sort function is for a _cb suffix, so `fu_util_sort_devices_by_flags_cb`